### PR TITLE
Action List: Close Actions Menu on Action click

### PR DESCRIPTION
**Pull request by ElliotTheHuman**
### :sparkles: Changes

- Updated `ActionListItemAction` to close Action menu on `MenuItem` click
- Changed "children" property in `ActionListColumn` to "title"

### :white_check_mark: Requirements

- [x] Build and tests are passing
- [x] PR is ideally < 400LOC

### :camera: Screenshots
menu-close-on-click

### CODE REVIEW

File: packages/components/src/ActionList/ActionListItem.tsx [hunk 1]
```diff
@@ -59,9 +59,15 @@ const ActionListItemInternal: FC<ActionListItemProps> = ({
     }
   }
 
+  const [isOpen, setOpen] = useState(false)
```
Comment (lukelooker) [inline packages/components/src/ActionList/ActionListItem.tsx]: What's this for?

File: packages/components/src/ActionList/ActionListItem.tsx [hunk 2]
```diff
@@ -59,9 +59,15 @@ const ActionListItemInternal: FC<ActionListItemProps> = ({
...
+
   const itemActions = actions && (
     <div onClick={handleMenuClick}>
-      <Menu hoverDisclosureRef={actionListItemRef}>
+      <Menu
+        isOpen={isOpen}
+        setOpen={setOpen}
```
Comment (lukelooker) [inline packages/components/src/ActionList/ActionListItem.tsx]: Not sure I understand what these are up to?

Comment (ElliotTheHuman) [reply packages/components/src/ActionList/ActionListItem.tsx]: Discussed in Slack, but this addition stemmed from my misunderstanding of `Menu`. I originally thought you needed to pass a `setOpen` into the `Menu` element in order to be able to explicitly control the opening / closing of the `Menu` in `MenuItem`'s. Didn't realize you get access to a `setOpen` callback via context regardless of what's passed into the `Menu`.

Removed in 381b0db

File: packages/components/src/ActionList/ActionListItem.tsx (hunk 1 above)
Comment (ElliotTheHuman) [reply packages/components/src/ActionList/ActionListItem.tsx]: Resolved in 381b0db!

**Pull request merged**